assets/js/header-footer.js

```diff
@@ -0,0 +1,31 @@
+/******************************************************************************
+FSWD:  Christopher B. Zenner
+Date:  02/15/2020
+File:  header-footer.js
+Ver.:  0.1.0 20200215
+
+This JS script renders the header and footer common to the three pages of the
+site at http://icn2you.com/fsf-basic-portfolio.
+******************************************************************************/
+
+$(function() {
+  $("header").load("assets/common/header.html", function(response, status, xhr) {
```

Comment:
Cool. jQuery is great, but old. It's generally only used for legacy code at my work. You could do this about 10 different ways in jQuery 😄 . Hopefully you also get to check out something like the [fetch api](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).

Something to consider here is that some bots that crawl your site can't read javascript, so they won't be able to load the header and footer. Nowadays this isn't nearly as much of a problem as it used to be, but it's something to consider for SEO purposes, etc.